_posts/2019-11-27-gettingstarted-2020/getting-started-2020.Rmd

```diff
@@ -0,0 +1,144 @@
+
+<aside>
+The [regression tutorial](https://tensorflow.rstudio.com/tutorials/beginners/basic-ml/tutorial_basic_regression/) now uses feature columns for convenient data preprocessing.
+</aside>
+
+This introduction to regression is complemented by a [tutorial on multi-class classification](https://tensorflow.rstudio.com/tutorials/beginners/basic-ml/tutorial_basic_classification/) using "Fashion MNIST". It is equally suited for a first encounter with Keras.
+
+A third tutorial to be mentioned in this section is dedicated to [text classification](https://tensorflow.rstudio.com/tutorials/beginners/basic-ml/tutorial_basic_text_classification/). Here too, there is a hidden gem in the current version that makes text preprocessing a lot easier: `layer_text_vectorization`, one of the brand new [Keras preprocessing layers](https://github.com/keras-team/governance/blob/master/rfcs/20190502-preprocessing-layers.md).^[In fact, it is that new that as of this writing, you will have to install the nightly build of TensorFlow -- as well as `tensorflow` from github -- to use it.] If you've used Keras for NLP before: No more messing with `text_tokenizer`!
+
+<aside>
+Check out the new text vectorization layer in the [text classification tutorial](https://tensorflow.rstudio.com/tutorials/beginners/basic-ml/tutorial_basic_text_classification/).
+</aside>
+
+The tutorials just mentioned are all nice introductions explaining code as well as concepts. What if you're familiar with the basic procedure and just need a quick reminder (or: something to quickly copy-paste from)? The ideal document to consult for those purposes is the [Overview](https://tensorflow.rstudio.com/tutorials/beginners/).
+
+Now -- knowledge how to build models is fine, but as in data science overall, there is no modeling without data.
+
+### Data ingestion and preprocessing
+
+Two detailed, end-to-end tutorials show how to load [csv data](https://tensorflow.rstudio.com/tutorials/beginners/load/load_csv/) and 
+[images](https://tensorflow.rstudio.com/tutorials/beginners/load/load_image/), respectively.
+
+In current Keras, two mechanisms are central to data preparation. One is the use of [tfdatasets pipelines](https://tensorflow.rstudio.com/guide/tfdatasets/introduction/). `tfdatasets` lets you load data in a streaming fashion (batch-by-batch), optionally applying transformations as you go. The other handy device here is [feature specs](https://tensorflow.rstudio.com/guide/tfdatasets/feature_spec/) resp. [feature columns](https://tensorflow.rstudio.com/guide/tfdatasets/feature_columns/). Together with a matching Keras layer, these allow for transforming the input data without having to think about what the new format will mean to Keras.
```

Comment:
OK, so I looked up [resp.](https://en.wiktionary.org/wiki/resp.) because I wasn't quite sure what it meant as you've used it (I thought it meant respectively, and it _does_ there's just a slight difference in order that I guess is the opposite in German). 

From the link above: 
> Like the unabbreviated form respectively, this abbreviation is used postfix, as in "X or Y, resp.". Some ESL writers use it infix, as in "X resp. Y", by false analogy with German bzw., dated-syntax Danish hhv. and Swedish resp., but this is not considered correct.

Reply:
So, you would use "resp." just as you've used "respectively" in the first sentence of the paragraph above, i.e. "...feature specs or feature columns, resp."

Reply:
Oh!! That's good to know, great you looked it up :-)
Does "ESL" stand for "English as second language" (yeah just googled , evidently it does).
Then I need sth else to fit in that position, syntactically ... 

actually these are pretty _wrong_  semantically even (or not _wrong_, but don't match) ... https://dict.leo.org/german-english/beziehungsweise

Reply:
I think here i can just throw it out completely, actually i'm not even using it the way i thought i was ;-)))